src/Core/src/Platform/iOS/ViewExtensions.cs

```diff
@@ -698,5 +698,115 @@ internal static void UpdateLayerBorder(this CoreAnimation.CALayer layer, IButton
 			if (stroke.CornerRadius >= 0)
 				layer.CornerRadius = stroke.CornerRadius;
 		}
+
+		internal static UIViewController? GetViewController<T>(this UIView view) where T : UIViewController
```

Comment:
Thoughts on `FindViewController`? `Get` feels like you'd only want the immediate VS for a particular View.  `SearchViewController`/ `FindFirstviewController`?